### ISSUE TITLE
Add completed checkmark for transcript to match the media component

### ISSUE THREAD

**Issue by lemmyadams** [cgkineo/adapt-youtube#33]
### Subject of the issue/enhancement/features
With the recent change to the media component to have a completed checkmark for the transcript buttons,
(see here: https://github.com/adaptlearning/adapt-contrib-media/issues/290)
The same should be done for the vimeo and youtube components as they are specialised versions of the media component.
This issue in particular focuses on the youtube component

### Expected behaviour
After interacting with the transcript button a check should appear

### Screenshots (if you can)
![image](https://github.com/user-attachments/assets/d883f827-7fcd-4cce-bf12-704a80b370bc)
